two_points_on_the_same_branch.sh

```diff
@@ -55,6 +73,45 @@ else
 fi
 
 working_dir="${WORKSPACE%@*}/$BUILD_ID"
+mkdir "$working_dir"
+
+log_dir=$working_dir/logs
+mkdir "$log_dir"
+
+if [ ! -z "${coq_pr_number}" ]; then
+    github_response="$(curl "https://api.github.com/repos/coq/coq/pulls/${coq_pr_number}")"
+    new_coq_repository="$(echo "${github_response}" | jq -r '.head.repo.clone_url')"
+    new_coq_commit="$(echo "${github_response}" | jq -r '.head.sha')"
+    old_coq_repository="$(echo "${github_response}" | jq -r '.base.repo.clone_url')"
+    old_coq_commit="$(echo "${github_response}" | jq -r '.base.sha')"
+    coq_pr_title="$(echo "${github_response}" | jq -r '.title')"
+
+    for val in "${new_coq_repository}" "${new_coq_commit}" "${old_coq_repository}" "${old_coq_commit}" "${coq_pr_title}"; do
+        if [ -z "$val" ] || [ "val" == "null" ]; then
+            echo 'ERROR: Invalid Response:' > /dev/stderr
+            echo "${github_response}" > /dev/stderr
+            echo "Info:" > /dev/stderr
+            curl -i "https://api.github.com/repos/coq/coq/pulls/${coq_pr_number}" > /dev/stderr
+            exit 1
+        fi
+    done
+
+    if [ -z "$BENCH_DEBUG" ]; then # if it's non-empty, this'll get
+                                   # printed later anyway.  But we
+                                   # want to see it always if we're
+                                   # automatically computing values
+        echo "DEBUG: new_coq_repository = $new_coq_repository"
+        echo "DEBUG: new_coq_commit = $new_coq_commit"
+        echo "DEBUG: old_coq_repository = $old_coq_repository"
+        echo "DEBUG: old_coq_commit = $old_coq_commit"
+    fi
+
+    # cf https://wiki.jenkins.io/display/JENKINS/Jenkins+CLI
+    # TODO: Is there a better place to put the cli?  Is it already available somewhere?
+    curl "${JENKINS_URL}/jnlpJars/jenkins-cli.jar" -o "$working_dir/jenkins-cli.jar"
```

Comment:
@maximedenes Is this (https://ci.inria.fr/coq/jnlpJars/jenkins-cli.jar) the same as the one that you pointed at that's at /home/jenkins/jenkins-slave/slave.jar